cpp/ql/lib/semmle/code/cpp/Concept.qll

```diff
@@ -153,11 +153,110 @@ class NestedRequirementExpr extends Expr, @nested_requirement {
 
 /**
  * A C++ concept id expression.
+ *
+ * For example, if:
+ * ```cpp
+ * template<typename T, T X> concept C = ...;
+ * ...
+ * requires { C<int, 1>; };
+ * ```
+ * then `C<int, 1>` is a concept id expression that refers to
+ * the concept `C`.
  */
 class ConceptIdExpr extends RequirementExpr, @concept_id {
-  override string toString() { result = "concept<...>" }
+  override string toString() {
+    exists(string name |
```

Comment:
Is this preferable to `result = this.getConcept().getName() + "<...>"`

Reply:
You're absolutely right, that's much better.

Reply:
Fixed.